src/Uno.UWP/Devices/Geolocation/Geolocator.Android.cs

```diff
@@ -125,9 +125,7 @@ public static Geoposition ToGeoPosition(this Location location)
 							Latitude = location.Latitude,
 							Longitude = location.Longitude,
 							Altitude = location.Altitude,
-						},
-						AltitudeReferenceSystem.Ellipsoid,
-						Wgs84SpatialReferenceId
+						}
```

Comment:
constant Wgs84SpatialReferenceId was used for which reason? 

Also I suppose you can remove the private constant as it's not used anymore.

Reply:
I don't know why it was added, but all it did was break the code.